ERCS/erc-draft-event-driven.md

```diff
@@ -0,0 +1,261 @@
+---
+eip: <to be assigned>
```

Comment:
```suggestion
eip: 7672
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.